### PR TITLE
SG-31340 Deprecation of Python 2

### PR DESCRIPTION
**Pull request by minna-feng**

### CODE REVIEW

File: tests/test_mockgun.py [hunk 1]
```diff
@@ -53,28 +53,6 @@
 )
 
 
-# FIXME: This should probably be refactored into a base class for
-# all test bases
-class TestBaseWithExceptionTests(unittest.TestCase):
```
Comment (carlos-villavicencio-adsk) [inline tests/test_mockgun.py:58]: Nice!

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM! Thank you for this :shipit:

**Comment by carlos-villavicencio-adsk** [comment]
Also, would you mind removing the Python 2 badge from the README.md file, please?

**Review by NorberMV — APPROVED** [review]
LGTM, Nice job!

**Review by NorberMV — APPROVED** [review]
LGTM, Good job!

**Pull request merged**